src/rprblender/export/material.py

```diff
@@ -90,7 +90,6 @@ def sync(rpr_context: RPRContext, material: bpy.types.Material, input_socket_key
     rpr_material = node_parser.final_export(input_socket_key)
 
     if rpr_material:
-        rpr_context.set_material_node_as_material(mat_key, rpr_material)
```

Comment:
@bnagirniak what is the reason for this?

Reply:
I removed this line because it is already exist below, It was duplicated in this block.